apps/docs/src/css/custom.css

```
@@ -4,14 +4,36 @@
+ * `@charset` / empty `@layer`) or the browser drops it silently.
  */
 @import './tokens.generated.css';
 
+/*
+ * Geist Sans + Geist Mono (OFL), self-hosted same-origin. Metric-matched
+ * pair from the Geist superfamily: leading the sans/mono token stacks
+ * with these (see apps/docs/tokens/font.json) is the root-cause fix for
+ * monospace rendering visibly larger than the proportional font.
+ */
+@font-face {
+  font-family: 'Geist';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)